internal/oidc/provider/manager/manager.go

```diff
@@ -134,6 +135,13 @@ func (m *Manager) SetProviders(federationDomains ...*provider.FederationDomainIs
 			oauthHelperWithKubeStorage,
 		)
 
+		m.providerHandlers[(issuerHostWithPath + oidc.PinnipedLoginPath)] = login.NewHandler(
+			upstreamStateEncoder,
+			csrfCookieEncoder,
+			login.NewGetHandler(incomingProvider.IssuerPath()+oidc.PinnipedLoginPath),
```

Comment:
```suggestion
			login.NewGetHandler(issuerHostWithPath + oidc.PinnipedLoginPath),
```
?

Reply:
Not sure I understand this suggestion. Why do you prefer the absolute path to the relative path?